.gitignore

```diff
@@ -10,3 +10,4 @@ data
 output
 # Docs folder used for papers etc
 analysis/docs
+node_modules
```

Comment:
kind of a nit, this should already be in /Planscape/src/interface/.gitignore file because the node_modules are only needed for the front end. if you have them installed in this higher level directory instead, you would need to add this line.

Reply:
Done